src/modules/workspace/promote.ts

```diff
@@ -10,26 +10,25 @@ import useCmd from './use'
 const { promote, get } = workspaces
 const [account, currentWorkspace] = [getAccount(), getWorkspace()]
 
-const isMaster = Promise.method((workspace: string) => {
+const isMaster = (workspace: string) => {
   if (workspace === 'master') {
     throw new CommandError(`It is not possible to promote workspace ${workspace} to master`)
   }
-})
-
-const isPromotable = (workspace: string) =>
-  isMaster(workspace).then(async () => {
-    const meta = await get(account, currentWorkspace)
-    if (!meta.production) {
-      throw new CommandError(
-        `Workspace ${chalk.green(currentWorkspace)} is not a ${chalk.green(
-          'production'
-        )} workspace\nOnly production workspaces may be promoted\nUse the command ${chalk.blue(
-          'vtex workspace create <workspace> --production'
-        )} to create a production workspace`
-      )
-    }
-  })
+}
 
+const isPromotable = async (workspace: string) => {
+  isMaster(workspace)
```

Comment:
no need to `await` this?

Reply:
`isMaster` just throws an error if the workspace is master, it's sync:
```ts
const isMaster = (workspace: string) => {
  if (workspace === 'master') {
    throw new CommandError(`It is not possible to promote workspace ${workspace} to master`)
  }
}
```